app/components/forms/admin/video-channel-form.ts

```diff
@@ -21,7 +21,7 @@ export default class VideoChannelForm extends Component<Args> {
     try {
       this.loading = true;
       await this.args.videoChannel.save();
-      this.router.transitionTo('admin.video-channels');
+      this.router.transitionTo('admin.video');
```

Comment:
after creating/editing video channel, this should go to `admin/video/channels` but currently going to `admin/video/recordings`

Reply:
Also add sorting start/end time.